app/views/distributions/_distribution.html.erb

```diff
@@ -0,0 +1,135 @@
+        <% else %>
+          openSUSE Tumbleweed
+        <% end %>
+      </h1>
+    </div>
+  </div>
+  <div class="border-bottom" role="tablist">
+    <ul class="container nav nav-tabs nav-justified border-0">
+      <% @yaml_data&.each do |tab| %>
+      
+      <li class="nav-item">
+        <a class="nav-link h-100<% if @yaml_data[0]["name"] == tab["name"] %> active<% end %>" data-toggle="tab" href="#<%= tab["name"] %>-ports" role="tab">
+          <%= _(tab["name"]) %>
+          <% tab["arches"]&.each do |arch| %>
+            <span class="badge badge-<%= arch["color"] %>"><%= arch["name"] %></span>
+          <% end %>
+        </a>
+      </li>
+      <% end %>
+    </ul>
+  </div>
+</header>
+
+<section class="mb-5">
+  <div class="container">
+    <div class="tab-content">
+      <% @yaml_data&.each do |tab| %>
+      <div class="tab-pane <% if @yaml_data[0]["name"] == tab["name"] %>active<% end %>" id="<%= tab["name"] %>-ports" role="tabpanel">
+        <% if tab["info"] %>
+          <div class="alert alert-<%= tab["info-colour"] %>">
+            <%= markdownify( _(tab["info"]) ) %>
+          </div>
+        <% end %>
+        <% tab["arches"]&.each do |arch| %>
+        <h3 class="my-3 text-center"><%= arch["name"] %></h3>
+        <div class="row">
+          <% arch["types"]&.each do |type| %>
+          <div class="col text-center">
+            <a href="<%= type["primary_link"] %>" class="btn btn-lg btn-block btn-<%= @colour %>">
+            <span class="typcn typcn-download-outline"></span>
+            <%= type["name"] %>
+            </a>
+            <div class="p-2">
+              <h5 class="mb-2 text-muted"><%= type["image_size"] %></h5>
+              <h6 class="mb-2 text-muted"><%= _(type["short"]) %></h6>
+              <p class="text-muted"><%= _(type["desc"]) %></p>
+
+              <% type["links"]&.each do |link| %>
+              <a href="<%= link["url"] %>" class="btn btn-link"><%= _(link["name"]) %></a>
+              <% end %>
+            </div>
+          </div>
+          <% end %>
+        </div>
+        <% end %>
+
+        <% if tab["choosing-media"] %>
+        <section class="my-3">
+          <div class="container" role="main">
+            <%= render "choosing_media" %>
+          </div>
+        </section>
+        <% end %>
+        <% if tab["leap-switch"] %>
+        <section class="my-3">
+          <div class="container" role="main">
+            <%= render "leap_switch" %>
+          </div>
+        </section>
+        <% end %>
+        <% if tab["tumbleweed-switch"] %>
+        <section class="my-3">
+          <div class="container" role="main">
+            <%= render "tumbleweed_switch" %>
+          </div>
+        </section>
+        <% end %>
+
+      </div><!-- /.tab-pane -->
+      <% end %>
+    </div><!-- /.tab-content -->
+    <% if @distro_type == "leap" %>
+      <% if @legacy_release && current_page?( :controller => 'distributions', :action => 'leap' ) %>
```

Comment:
do not leave spaces around parenthesis and use new ruby syntax for hashes. Robocop would complain if he could :rofl:

Reply:
Rubocop rules probably need updating to be harsher, I'm doing too good of a work avoiding it complaining

Reply:
I think you didn't get what I meant. `erb` files are not inspected by Rubocop, only `rb`. That why there was not offenses here. But they should be fixed to be coherent with the rest of the code.